CMakeLists.txt

```diff
@@ -273,7 +273,7 @@ if(MSVC)
   # XXX: /W4 gives too many warnings. #3241
   add_compile_options(/W3)
   add_definitions(-D_CRT_SECURE_NO_WARNINGS -D_CRT_NONSTDC_NO_DEPRECATE)
-  add_definitions(-DWIN32)
+  add_definitions(-DMSWIN)
```

Comment:
Also defined this for MinGW?

Reply:
Hmmm, no that doesn't sound right. I think this is only relevant for the MSVC compiler.

Reply:
Hmm, I'll need to test with MINGW to be sure though.

Reply:
Nope, you're right. Thanks for the help!